src/compiler/diagnosticMessages.json

```diff
@@ -3814,6 +3814,10 @@
         "category": "Suggestion",
         "code": 80001
     },
+    "Function may be converted to a class.": {

```

Comment:
@DanielRosenwasser any thoughts about the error message

Reply:
I think this message can be enhanced, @DanielRosenwasser might have better suggestions.

Reply:
We should refine this over time, but

```
This constructor function may be converted to a class declaration.
```

or

```
Constructor function may be better represented as a class declaration.
```